machine_management/applying-autoscaling.adoc

```diff
@@ -32,9 +32,43 @@ include::modules/cluster-autoscaler-cr.adoc[leveloffset=+2]
 :FeatureResourceName: ClusterAutoscaler
 include::modules/deploying-resource.adoc[leveloffset=+2]
 
-== Next steps
+//
+// Addresses WINC-551 - https://github.com/openshift/openshift-docs/pull/32773 - Delete before merge
+//
 
-* After you configure the cluster autoscaler, you must configure at least one machine autoscaler.
+[id="cluster-autoscaler-machineset"]
```

Comment:
Not sure if this level of detail is required. Just calling out that this also works for Windows nodes should be sufficient. We can also mention this in the 2.0.1 release notes.